apps/backend/src/modules/onboarding/libs/exceptions/exceptions.ts

```diff
@@ -0,0 +1 @@
+export { QuizError } from "shared";
```

Comment:
```suggestion
export { OnboardingError } from "shared";
```
Let's keep Onboarding and Quiz modules separately